packages/origin-backend/migrations/1588836124586-UserStatusAndAutoPublish.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Status1588836124586 implements MigrationInterface {
+    name = 'Status1588836124586';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "user" DROP COLUMN "autoPublish"`, undefined);
+        await queryRunner.query(`ALTER TABLE "user" ALTER COLUMN "status" DROP DEFAULT`, undefined);
```

Comment:
There should be default set. The fix should add that default option in to entity.ts file, so the typeorm migration tool will see that.